.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
           command: |
```

Comment:
I expect this CI config will have a provisioning job setup_mainnet or similar?
App deployment strategy to mainnet env to be decided?

Reply:
I plan to release manually, at least for now.